tests/parametric/test_telemetry.py

```diff
@@ -11,6 +11,26 @@
 from utils import context, scenarios, rfc, features
 
 
+telemetry_name_mapping = {
+    "trace_sample_rate": {"dotnet": "TODO",},
```

Comment:
@kevingosse can you fill these in for .NET? 🙏

Reply:
Done in https://github.com/DataDog/system-tests/pull/2172/commits/0f6af6416d1169f9541afad1888aeb6afc7343cd